geopandas/array.py

```diff
@@ -70,7 +71,7 @@ def _check_crs(left, right, allow_none=False):
     return True
 
 
-def _crs_mismatch_warn(left, right, stacklevel=3):
+def _crs_mismatch_warn(left, right, stacklevel: int = 3) -> None:
```

Comment:
if the return is `None`, there shouldn't add it.

Reply:
`-> None` just implies that the method is void. Shouldn't make a difference,, except for style if that's against our preferences.

Reply:
Python default return is `None`, unless `None` as an `Optional`.